### ISSUE TITLE
redundant "_version" attribute.

### ISSUE THREAD

**Issue by redmitry** [inab/benchmarking-data-model#14]
The version is explicitly stated within the schema attribute.

**Comment by jlgelpi** [comment]
Agree. But I would remove the _schema ( = id + _version ), as it takes a lot of extra disk space.

**Comment by jmfernandez** [comment]
I disagree removing _schema. It was introduced in order to give each schema a unique identifier in a single field, and removing _type + _version attributes. If we remove it, how do we know the right JSON schema?

I have some proposals. The first is having shorter _schema values: we can use the URIs we prefer, as far as they match the _id from the JSON schema. The other is at the database level: you could store a shorter, indexed version of it in a separate place.

But I would not drop _schema attribute for public documents.

**Comment by jlgelpi** [comment]
Ok then, lets remove _type and _version. The Base of all URIs in the schema is the same, so it can be easily replaced by a prefix/namespace. I'll add an internal resolution service to the API (that eventually can use identifiers.org)

**Comment by redmitry** [comment]
The "id" ("$id" in draft 5) MUST uniquely identify Json Schema and SHOULD be resolved to the Json Schema document.
Root Schema "id" MUST be absolute and MUST have no local parts (#xxx).
Subschemas MAY have they own RELATIVE "id" defined, so the resolved "id" is always a valid URI.

example:
```json
{
    "id": "https://www.elixir-europe.org/excelerate/WP2/json-schemas/0.3/Tool",
    "type": "object",
    "properties": {
        "A": {
            "id": "#A",
            "type": "string"
        }
    }
```
Here subschema for the property "A" is "https://www.elixir-europe.org/excelerate/WP2/json-schemas/0.3/Tool#A"

Although usually there is no reason to put "id" for subschemas, local definitions ("$ref") are resolved in a context of current "id" - in case there is no any "id" defined in subschemas all of them are in a context of the root "id".

IMO there is no need in "_schema".

**Comment by jlgelpi** [comment]
I think that the idea of _schema was to have a pointer to the schema within the data, so we can validate a standalone document.

**Comment by redmitry** [comment]
OIC. That's true. Unlike XML ( ;-) ) Json has no standard means to refer schema document.
In my API I use "_type" field for the purpose.

**Comment by jmfernandez** [comment]
I'm removing _type and _version fields from the schemas

**Comment by jmfernandez** [comment]
Fixed in 68d19b7df9e215455175c4422ddf38c6f779801b , 00993a3f7329e334c1e1ff5b5ad7abd0daf5d45b , 37cd745be40760e4e5c2c2bdfc0447dd9a3caf92 and 96f5e884d5a7a84a879b6ae7bc2c8c431d9340bb commits
